crates/sui-protocol-config/src/lib.rs

```diff
@@ -1188,8 +1188,18 @@ impl ProtocolConfig {
     /// Get the value ProtocolConfig that are in effect during the given protocol version.
     pub fn get_for_version(version: ProtocolVersion, chain: Chain) -> Self {
         // ProtocolVersion can be deserialized so we need to check it here as well.
-        assert!(version.0 >= ProtocolVersion::MIN.0, "{:?}", version);
-        assert!(version.0 <= ProtocolVersion::MAX_ALLOWED.0, "{:?}", version);
+        assert!(
+            version.0 >= ProtocolVersion::MIN.0,
+            "Requested version {:?} must be equal or bigger than the minimum supported version {:?}",
```

Comment:
If this is the error operators will see, maybe mention "on-chain version" vs "local binary version" to be more informative.